.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so we can reproduce?
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: os-name
+    attributes:
+      label: OS Name
+      description: Which Operating System are you using?
```

Comment:
For consistency, I'd go with "operating system" (lowercase). Using OS is valid as well.